app/uploaders/profile_photo_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class ProfilePhotoUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
```

Comment:
this is fine, but as soon as this is merged you should move to "Cloud Storage" article immediately. otherwise your app is gonna have a bunch of junk photos in github